README.md

```diff
@@ -57,17 +59,37 @@ content-length: 98
 content-type: application/json
 
 {
-    "name": "ut dolore nulla non ex",
+    "name": "doggie",
     "photoUrls": [
-        "aute sed in qui",
-        "magna Excepteur",
-        "tempor dolor"
-    ]
+        "fugiat",
+        "in amet"
+    ],
+    "id": 38621518,
+    "category": {
+        "id": -77973282,
+        "name": "nisi"
+    },
+    "tags": [
+        {"id": 92329113, "name": "nulla"},
+        {"id": -72921961, "name": "esse"},
+        {"id": 38941757, "name": "velit incididunt quis ullamco magna"},
+        {"id": -14316986, "name": "occaecat exercitation cillum"}
+    ],
+    "status": "available"
```

Comment:
On a different topic, this example is too long

Reply:
So here are the options:
- try to be consistent with real example <-- this is how it looks currently
- shorten example
- modify petstore

I vote on first